ts/src/probit.ts

```diff
@@ -414,7 +414,22 @@ export default class probit extends Exchange {
             const name = this.safeString (displayName, 'en-us');
             const platforms = this.safeValue (currency, 'platform', []);
             const platformsByPriority = this.sortBy (platforms, 'priority');
-            const platform = this.safeValue (platformsByPriority, 0, {});
+            let platform = undefined;
+            for (let j = 0; j < platformsByPriority.length; i++) {
```

Comment:
since you're already iterating the networks I think we can go the extra mile and add parse them